src/address_space/backend/linear.rs

```diff
@@ -1,19 +1,18 @@
-use memory_addr::{PhysAddr, VirtAddr};
-use page_table_multiarch::MappingFlags;
-use page_table_multiarch::PagingHandler;
+use memory_addr::PhysAddr;
+use page_table_multiarch::{MappingFlags, PagingHandler};
 
-use crate::backend::Backend;
-use crate::npt::NestedPageTable as PageTable;
+use super::Backend;
+use crate::{npt::NestedPageTable as PageTable, GuestPhysAddr};
 
-impl Backend {
+impl<H: PagingHandler> Backend<H> {
```

Comment:
why re-imported generic type `PagingHandler` here?
is it used for `Backend`?

Reply:
To implement `MappingBackend`. See `src/address_space/backend/mod.rs:54`.